nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/pom.xml

```diff
@@ -20,8 +20,8 @@ language governing permissions and limitations under the License. -->
     <packaging>jar</packaging>
 
     <properties>
-        <snmp4j.version>1.10.1</snmp4j.version>
-        <snmp4j-agent.version>1.3.1</snmp4j-agent.version>
+        <snmp4j.version>2.5.6</snmp4j.version>
+        <snmp4j-agent.version>2.5.3</snmp4j-agent.version>
```

Comment:
I noticed as I was checking LICENSE and NOTICE that there was a newer version of this (2.6.0). Curious if you considered the new version.